<?xml version="1.0" encoding="UTF-8" ?>
<rss version="2.0" xmlns:atom="http://www.w3.org/2005/Atom" xmlns:dc="http://purl.org/dc/elements/1.1/">
  <channel>
    <title>Error trying to run client</title>
    <link>https://forum.moparisthebest.com/t/error-trying-to-run-client/432319</link>
    <description>[Cache] Loading cache...
[Cache] Loaded cache!
[MapXTEA] Loaded 1502 map XTEA key(s)
[PacketManager] Loading packet handlers...
[PacketManager] Loaded 60 packet handlers.
com.mysql.jdbc.CommunicationsException: Communications link failure due to under
lying exception:

** BEGIN NESTED EXCEPTION **

java.net.ConnectException
MESSAGE: Connection timed out: connect

STACKTRACE:

java.net.ConnectException: Connection timed out: connect
        at java.net.TwoStacksPlainSocketImpl.socketConnect(Native Method)
        at java.net.AbstractPlainSocketImpl.doConnect(Unknown Source)
        at java.net.AbstractPlainSocketImpl.connectToAddress(Unknown Source)
        at java.net.AbstractPlainSocketImpl.connect(Unknown Source)
        at java.net.PlainSocketImpl.connect(Unknown Source)
        at java.net.SocksSocketImpl.connect(Unknown Source)
        at java.net.Socket.connect(Unknown Source)
        at java.net.Socket.connect(Unknown Source)
        at java.net.Socket.&lt;init&gt;(Unknown Source)
        at java.net.Socket.&lt;init&gt;(Unknown Source)
        at com.mysql.jdbc.StandardSocketFactory.connect(StandardSocketFactory.ja
va:173)
        at com.mysql.jdbc.MysqlIO.&lt;init&gt;(MysqlIO.java:267)
        at com.mysql.jdbc.Connection.createNewIO(Connection.java:2739)
        at com.mysql.jdbc.Connection.&lt;init&gt;(Connection.java:1553)
        at com.mysql.jdbc.NonRegisteringDriver.connect(NonRegisteringDriver.java
:266)
        at java.sql.DriverManager.getConnection(Unknown Source)
        at java.sql.DriverManager.getConnection(Unknown Source)
        at org.dementhium.mysql.DatabaseManager.establishConnection(DatabaseMana
ger.java:44)
        at org.dementhium.mysql.impl.PlayerLoader.&lt;init&gt;(PlayerLoader.java:49)
        at org.dementhium.model.World.&lt;init&gt;(World.java:95)
        at org.dementhium.model.World.&lt;clinit&gt;(World.java:72)
        at org.dementhium.RS2ServerBootstrap.main(RS2ServerBootstrap.java:56)


** END NESTED EXCEPTION **



Last packet sent to the server was 47 ms ago.
        at com.mysql.jdbc.Connection.createNewIO(Connection.java:2814)
        at com.mysql.jdbc.Connection.&lt;init&gt;(Connection.java:1553)
        at com.mysql.jdbc.NonRegisteringDriver.connect(NonRegisteringDriver.java
:266)
        at java.sql.DriverManager.getConnection(Unknown Source)
        at java.sql.DriverManager.getConnection(Unknown Source)
        at org.dementhium.mysql.DatabaseManager.establishConnection(DatabaseMana
ger.java:44)
        at org.dementhium.mysql.impl.PlayerLoader.&lt;init&gt;(PlayerLoader.java:49)
        at org.dementhium.model.World.&lt;init&gt;(World.java:95)
        at org.dementhium.model.World.&lt;clinit&gt;(World.java:72)
        at org.dementhium.RS2ServerBootstrap.main(RS2ServerBootstrap.java:56)
com.mysql.jdbc.CommunicationsException: Communications link failure due to under
lying exception:

** BEGIN NESTED EXCEPTION **

java.net.ConnectException
MESSAGE: Connection timed out: connect

STACKTRACE:

java.net.ConnectException: Connection timed out: connect
        at java.net.TwoStacksPlainSocketImpl.socketConnect(Native Method)
        at java.net.AbstractPlainSocketImpl.doConnect(Unknown Source)
        at java.net.AbstractPlainSocketImpl.connectToAddress(Unknown Source)
        at java.net.AbstractPlainSocketImpl.connect(Unknown Source)
        at java.net.PlainSocketImpl.connect(Unknown Source)
        at java.net.SocksSocketImpl.connect(Unknown Source)
        at java.net.Socket.connect(Unknown Source)
        at java.net.Socket.connect(Unknown Source)
        at java.net.Socket.&lt;init&gt;(Unknown Source)
        at java.net.Socket.&lt;init&gt;(Unknown Source)
        at com.mysql.jdbc.StandardSocketFactory.connect(StandardSocketFactory.ja
va:173)
        at com.mysql.jdbc.MysqlIO.&lt;init&gt;(MysqlIO.java:267)
        at com.mysql.jdbc.Connection.createNewIO(Connection.java:2739)
        at com.mysql.jdbc.Connection.&lt;init&gt;(Connection.java:1553)
        at com.mysql.jdbc.NonRegisteringDriver.connect(NonRegisteringDriver.java
:266)
        at java.sql.DriverManager.getConnection(Unknown Source)
        at java.sql.DriverManager.getConnection(Unknown Source)
        at org.dementhium.mysql.DatabaseManager.establishConnection(DatabaseMana
ger.java:44)
        at org.dementhium.mysql.impl.Highscores.&lt;init&gt;(Highscores.java:24)
        at org.dementhium.model.World.&lt;init&gt;(World.java:96)
        at org.dementhium.model.World.&lt;clinit&gt;(World.java:72)
        at org.dementhium.RS2ServerBootstrap.main(RS2ServerBootstrap.java:56)


** END NESTED EXCEPTION **



Last packet sent to the server was 0 ms ago.
        at com.mysql.jdbc.Connection.createNewIO(Connection.java:2814)
        at com.mysql.jdbc.Connection.&lt;init&gt;(Connection.java:1553)
        at com.mysql.jdbc.NonRegisteringDriver.connect(NonRegisteringDriver.java
:266)
        at java.sql.DriverManager.getConnection(Unknown Source)
        at java.sql.DriverManager.getConnection(Unknown Source)
        at org.dementhium.mysql.DatabaseManager.establishConnection(DatabaseMana
ger.java:44)
        at org.dementhium.mysql.impl.Highscores.&lt;init&gt;(Highscores.java:24)
        at org.dementhium.model.World.&lt;init&gt;(World.java:96)
        at org.dementhium.model.World.&lt;clinit&gt;(World.java:72)
        at org.dementhium.RS2ServerBootstrap.main(RS2ServerBootstrap.java:56)
[World] Loading World...
[ClanManager] Loading clans....
[ClanManager] Loaded 42 clans.
[AreaManager] Loading areas...
[AreaManager] Loaded 48 areas.
[NPCDropLoader] Loading drops...
[NPCDropLoader] Loaded 2649 drops
[SpellContainer] Loading magic spells...
[SpellContainer] Loaded 43 magic spell handlers.
[SpecialAttackContainer] Loading special attacks...
[SpecialAttackContainer] Loaded 37 special attacks.
[DialogueManager] Loading dialogues...
[DialogueManager] Loaded 35 dialogues.
[DialogueManager] Loading NPC dialogue ids...
[DialogueManager] Loaded 1 NPC dialogue ids.
[ShopManager] Loaded 24 Shops
[ObjectManager] Loading objects...
[ObjectManager] Loaded 13 objects.
[ItemDefinition] Loading item definitions...
[ItemDefinition] Loaded 20430 item definitions.
[RangeWeapon] Loading range weapon info...
[RangeWeapon] Loaded 184 range weapon definitions.
[Ammunition] Loading range ammunition info...
[Ammunition] Loaded 219 ammunition definitions.
[NPCDefinitionPacker] Packing NPC definitions.
java.io.IOException: The system cannot find the path specified
        at java.io.WinNTFileSystem.createFileExclusively(Native Method)
        at java.io.File.createNewFile(Unknown Source)
        at org.dementhium.tools.converters.NPCDefinitionPacker.pack(NPCDefinitio
nPacker.java:47)
        at org.dementhium.model.definition.NPCDefinition.init(NPCDefinition.java
:43)
        at org.dementhium.model.World.load(World.java:127)
        at org.dementhium.RS2ServerBootstrap.main(RS2ServerBootstrap.java:56)
[NPCDefinitionPacker] Packed 0 functional npc definitions.
[NPCDefinition] Loading npc definitions...
Exception in thread &quot;main&quot; java.io.FileNotFoundException: C:\Documents and Setti
ngs\Owner\My Documents\sever\downloads\NDE\NPCDefinitions.bin\NDE\NPCDefinitions
.bin\. (The system cannot find the path specified)
        at java.io.RandomAccessFile.open(Native Method)
        at java.io.RandomAccessFile.&lt;init&gt;(Unknown Source)
        at java.io.RandomAccessFile.&lt;init&gt;(Unknown Source)
        at org.dementhium.model.definition.NPCDefinition.init(NPCDefinition.java
:48)
        at org.dementhium.model.World.load(World.java:127)
        at org.dementhium.RS2ServerBootstrap.main(RS2ServerBootstrap.java:56)

any1 have any idee how to fix&gt;?</description>
    
    <lastBuildDate>Thu, 09 Feb 2012 14:59:04 +0000</lastBuildDate>
    <category>Runescape</category>
    <atom:link href="https://forum.moparisthebest.com/t/error-trying-to-run-client/432319.rss" rel="self" type="application/rss+xml" />
      <item>
        <title>Error trying to run client</title>
        <dc:creator><![CDATA[@RiiPiiNFtW RiiPiiNFtW]]></dc:creator>
        <description><![CDATA[
          <p><a href="https://forum.moparisthebest.com/u/riipiinftw">@RiiPiiNFtW</a> wrote:</p>
          <blockquote>
              <p>Stop depleting this</p>
          </blockquote>
          <p><a href="https://forum.moparisthebest.com/t/error-trying-to-run-client/432319/4">Read full topic</a></p>
        ]]></description>
        <link>https://forum.moparisthebest.com/t/error-trying-to-run-client/432319/4</link>
        <pubDate>Thu, 09 Feb 2012 14:59:04 +0000</pubDate>
        <guid isPermaLink="false">forum.moparisthebest.com-post-432319-4</guid>
        <source url="https://forum.moparisthebest.com/t/error-trying-to-run-client/432319.rss">Error trying to run client</source>
      </item>
      <item>
        <title>Error trying to run client</title>
        <dc:creator><![CDATA[@doomboy907 doomboy907]]></dc:creator>
        <description><![CDATA[
          <p><a href="https://forum.moparisthebest.com/u/doomboy907">@doomboy907</a> wrote:</p>
          <blockquote>
              <p>srry kinda a noob how do i fix it got mysql on no idee how to use tho  <img src="https://forum.moparisthebest.com/images/emoji/twitter/confused.png?v=5" title=":confused:" class="emoji" alt=":confused:"></p>
          </blockquote>
          <p><a href="https://forum.moparisthebest.com/t/error-trying-to-run-client/432319/3">Read full topic</a></p>
        ]]></description>
        <link>https://forum.moparisthebest.com/t/error-trying-to-run-client/432319/3</link>
        <pubDate>Thu, 09 Feb 2012 13:17:25 +0000</pubDate>
        <guid isPermaLink="false">forum.moparisthebest.com-post-432319-3</guid>
        <source url="https://forum.moparisthebest.com/t/error-trying-to-run-client/432319.rss">Error trying to run client</source>
      </item>
      <item>
        <title>Error trying to run client</title>
        <dc:creator><![CDATA[@C_Imbued C++ Imbued]]></dc:creator>
        <description><![CDATA[
          <p><a href="https://forum.moparisthebest.com/u/c_imbued">@C_Imbued</a> wrote:</p>
          <blockquote>
              <p><code>com.mysql.jdbc.CommunicationsException: Communications link failure due to under
lying exception:</code></p>
<pre><code class="lang-auto"></code></pre>
<p>Links broken and due to the link being broken it cannot connect ??</p>
          </blockquote>
          <p><a href="https://forum.moparisthebest.com/t/error-trying-to-run-client/432319/2">Read full topic</a></p>
        ]]></description>
        <link>https://forum.moparisthebest.com/t/error-trying-to-run-client/432319/2</link>
        <pubDate>Thu, 09 Feb 2012 13:08:14 +0000</pubDate>
        <guid isPermaLink="false">forum.moparisthebest.com-post-432319-2</guid>
        <source url="https://forum.moparisthebest.com/t/error-trying-to-run-client/432319.rss">Error trying to run client</source>
      </item>
      <item>
        <title>Error trying to run client</title>
        <dc:creator><![CDATA[@doomboy907 doomboy907]]></dc:creator>
        <description><![CDATA[
          <p><a href="https://forum.moparisthebest.com/u/doomboy907">@doomboy907</a> wrote:</p>
          <blockquote>
              <p>[Cache] Loading cache…<br>
[Cache] Loaded cache!<br>
[MapXTEA] Loaded 1502 map XTEA key(s)<br>
[PacketManager] Loading packet handlers…<br>
[PacketManager] Loaded 60 packet handlers.<br>
com.mysql.jdbc.CommunicationsException: Communications link failure due to under<br>
lying exception:</p>
<p>** BEGIN NESTED EXCEPTION **</p>
<p>java.net.ConnectException<br>
MESSAGE: Connection timed out: connect</p>
<p>STACKTRACE:</p>
<p>java.net.ConnectException: Connection timed out: connect<br>
at java.net.TwoStacksPlainSocketImpl.socketConnect(Native Method)<br>
at java.net.AbstractPlainSocketImpl.doConnect(Unknown Source)<br>
at java.net.AbstractPlainSocketImpl.connectToAddress(Unknown Source)<br>
at java.net.AbstractPlainSocketImpl.connect(Unknown Source)<br>
at java.net.PlainSocketImpl.connect(Unknown Source)<br>
at java.net.SocksSocketImpl.connect(Unknown Source)<br>
at java.net.Socket.connect(Unknown Source)<br>
at java.net.Socket.connect(Unknown Source)<br>
at java.net.Socket.(Unknown Source)<br>
at java.net.Socket.(Unknown Source)<br>
at com.mysql.jdbc.StandardSocketFactory.connect(StandardSocketFactory.ja<br>
va:173)<br>
at com.mysql.jdbc.MysqlIO.(MysqlIO.java:267)<br>
at com.mysql.jdbc.Connection.createNewIO(Connection.java:2739)<br>
at com.mysql.jdbc.Connection.(Connection.java:1553)<br>
at com.mysql.jdbc.NonRegisteringDriver.connect(NonRegisteringDriver.java<br>
:266)<br>
at java.sql.DriverManager.getConnection(Unknown Source)<br>
at java.sql.DriverManager.getConnection(Unknown Source)<br>
at org.dementhium.mysql.DatabaseManager.establishConnection(DatabaseMana<br>
ger.java:44)<br>
at org.dementhium.mysql.impl.PlayerLoader.(PlayerLoader.java:49)<br>
at org.dementhium.model.World.(World.java:95)<br>
at org.dementhium.model.World.(World.java:72)<br>
at org.dementhium.RS2ServerBootstrap.main(RS2ServerBootstrap.java:56)</p>
<p>** END NESTED EXCEPTION **</p>
<p>Last packet sent to the server was 47 ms ago.<br>
at com.mysql.jdbc.Connection.createNewIO(Connection.java:2814)<br>
at com.mysql.jdbc.Connection.(Connection.java:1553)<br>
at com.mysql.jdbc.NonRegisteringDriver.connect(NonRegisteringDriver.java<br>
:266)<br>
at java.sql.DriverManager.getConnection(Unknown Source)<br>
at java.sql.DriverManager.getConnection(Unknown Source)<br>
at org.dementhium.mysql.DatabaseManager.establishConnection(DatabaseMana<br>
ger.java:44)<br>
at org.dementhium.mysql.impl.PlayerLoader.(PlayerLoader.java:49)<br>
at org.dementhium.model.World.(World.java:95)<br>
at org.dementhium.model.World.(World.java:72)<br>
at org.dementhium.RS2ServerBootstrap.main(RS2ServerBootstrap.java:56)<br>
com.mysql.jdbc.CommunicationsException: Communications link failure due to under<br>
lying exception:</p>
<p>** BEGIN NESTED EXCEPTION **</p>
<p>java.net.ConnectException<br>
MESSAGE: Connection timed out: connect</p>
<p>STACKTRACE:</p>
<p>java.net.ConnectException: Connection timed out: connect<br>
at java.net.TwoStacksPlainSocketImpl.socketConnect(Native Method)<br>
at java.net.AbstractPlainSocketImpl.doConnect(Unknown Source)<br>
at java.net.AbstractPlainSocketImpl.connectToAddress(Unknown Source)<br>
at java.net.AbstractPlainSocketImpl.connect(Unknown Source)<br>
at java.net.PlainSocketImpl.connect(Unknown Source)<br>
at java.net.SocksSocketImpl.connect(Unknown Source)<br>
at java.net.Socket.connect(Unknown Source)<br>
at java.net.Socket.connect(Unknown Source)<br>
at java.net.Socket.(Unknown Source)<br>
at java.net.Socket.(Unknown Source)<br>
at com.mysql.jdbc.StandardSocketFactory.connect(StandardSocketFactory.ja<br>
va:173)<br>
at com.mysql.jdbc.MysqlIO.(MysqlIO.java:267)<br>
at com.mysql.jdbc.Connection.createNewIO(Connection.java:2739)<br>
at com.mysql.jdbc.Connection.(Connection.java:1553)<br>
at com.mysql.jdbc.NonRegisteringDriver.connect(NonRegisteringDriver.java<br>
:266)<br>
at java.sql.DriverManager.getConnection(Unknown Source)<br>
at java.sql.DriverManager.getConnection(Unknown Source)<br>
at org.dementhium.mysql.DatabaseManager.establishConnection(DatabaseMana<br>
ger.java:44)<br>
at org.dementhium.mysql.impl.Highscores.(Highscores.java:24)<br>
at org.dementhium.model.World.(World.java:96)<br>
at org.dementhium.model.World.(World.java:72)<br>
at org.dementhium.RS2ServerBootstrap.main(RS2ServerBootstrap.java:56)</p>
<p>** END NESTED EXCEPTION **</p>
<p>Last packet sent to the server was 0 ms ago.<br>
at com.mysql.jdbc.Connection.createNewIO(Connection.java:2814)<br>
at com.mysql.jdbc.Connection.(Connection.java:1553)<br>
at com.mysql.jdbc.NonRegisteringDriver.connect(NonRegisteringDriver.java<br>
:266)<br>
at java.sql.DriverManager.getConnection(Unknown Source)<br>
at java.sql.DriverManager.getConnection(Unknown Source)<br>
at org.dementhium.mysql.DatabaseManager.establishConnection(DatabaseMana<br>
ger.java:44)<br>
at org.dementhium.mysql.impl.Highscores.(Highscores.java:24)<br>
at org.dementhium.model.World.(World.java:96)<br>
at org.dementhium.model.World.(World.java:72)<br>
at org.dementhium.RS2ServerBootstrap.main(RS2ServerBootstrap.java:56)<br>
[World] Loading World…<br>
[ClanManager] Loading clans…<br>
[ClanManager] Loaded 42 clans.<br>
[AreaManager] Loading areas…<br>
[AreaManager] Loaded 48 areas.<br>
[NPCDropLoader] Loading drops…<br>
[NPCDropLoader] Loaded 2649 drops<br>
[SpellContainer] Loading magic spells…<br>
[SpellContainer] Loaded 43 magic spell handlers.<br>
[SpecialAttackContainer] Loading special attacks…<br>
[SpecialAttackContainer] Loaded 37 special attacks.<br>
[DialogueManager] Loading dialogues…<br>
[DialogueManager] Loaded 35 dialogues.<br>
[DialogueManager] Loading NPC dialogue ids…<br>
[DialogueManager] Loaded 1 NPC dialogue ids.<br>
[ShopManager] Loaded 24 Shops<br>
[ObjectManager] Loading objects…<br>
[ObjectManager] Loaded 13 objects.<br>
[ItemDefinition] Loading item definitions…<br>
[ItemDefinition] Loaded 20430 item definitions.<br>
[RangeWeapon] Loading range weapon info…<br>
[RangeWeapon] Loaded 184 range weapon definitions.<br>
[Ammunition] Loading range ammunition info…<br>
[Ammunition] Loaded 219 ammunition definitions.<br>
[NPCDefinitionPacker] Packing NPC definitions.<br>
java.io.IOException: The system cannot find the path specified<br>
at java.io.WinNTFileSystem.createFileExclusively(Native Method)<br>
at java.io.File.createNewFile(Unknown Source)<br>
at org.dementhium.tools.converters.NPCDefinitionPacker.pack(NPCDefinitio<br>
nPacker.java:47)<br>
at org.dementhium.model.definition.NPCDefinition.init(NPCDefinition.java<br>
:43)<br>
at org.dementhium.model.World.load(World.java:127)<br>
at org.dementhium.RS2ServerBootstrap.main(RS2ServerBootstrap.java:56)<br>
[NPCDefinitionPacker] Packed 0 functional npc definitions.<br>
[NPCDefinition] Loading npc definitions…<br>
Exception in thread “main” java.io.FileNotFoundException: C:\Documents and Setti<br>
ngs\Owner\My Documents\sever\downloads\NDE\NPCDefinitions.bin\NDE\NPCDefinitions<br>
.bin. (The system cannot find the path specified)<br>
at java.io.RandomAccessFile.open(Native Method)<br>
at java.io.RandomAccessFile.(Unknown Source)<br>
at java.io.RandomAccessFile.(Unknown Source)<br>
at org.dementhium.model.definition.NPCDefinition.init(NPCDefinition.java<br>
:48)<br>
at org.dementhium.model.World.load(World.java:127)<br>
at org.dementhium.RS2ServerBootstrap.main(RS2ServerBootstrap.java:56)</p>
<p>any1 have any idee how to fix&gt;?</p>
          </blockquote>
          <p><a href="https://forum.moparisthebest.com/t/error-trying-to-run-client/432319/1">Read full topic</a></p>
        ]]></description>
        <link>https://forum.moparisthebest.com/t/error-trying-to-run-client/432319/1</link>
        <pubDate>Thu, 09 Feb 2012 12:56:43 +0000</pubDate>
        <guid isPermaLink="false">forum.moparisthebest.com-post-432319-1</guid>
        <source url="https://forum.moparisthebest.com/t/error-trying-to-run-client/432319.rss">Error trying to run client</source>
      </item>
  </channel>
</rss>
